fe/fe-core/src/main/java/org/apache/doris/common/proc/AuthProcDir.java

```diff
@@ -31,7 +31,7 @@
  */
 public class AuthProcDir implements ProcDirInterface {
     public static final ImmutableList<String> TITLE_NAMES = new ImmutableList.Builder<String>()
-            .add("UserIdentity").add("Password").add("GlobalPrivs").add("CatalogPrivs")
+            .add("UserIdentity").add("Password").add("Roles").add("GlobalPrivs").add("CatalogPrivs")
```

Comment:
Is it suitable to add a new column here?
how about move it to `show roles`?

Reply:
Show roles is to show which users have roles assigned to them. Here is to show which roles users have